### DISCUSSION TITLE
`Workflow_job.completed` webhook: incorrect data in `steps` object

### DISCUSSION THREAD

**Discussion by Ari-Kishon** [community/community#29934] [Apps, API and Webhooks]
Hello,

Im trying to use the "steps" key in extract the running time of each step and get a total duration for each job.

the problem is that the resulting "steps" object seems to represent a workflow still in progress, despite the workflows status being "completed"

Attached below is a slightly redacted version of the problematic payload (_replaced sensitive fields with "---"_)
```
  "action": "completed",
  "workflow_job": {
    "id": ---,
    "run_id": --- ,
    "run_url": ---,
    "run_attempt": 1,
    "node_id": "---",
    "head_sha": "---",
    "url": "---",
    "html_url": "---",
    "status": "completed",
    "conclusion": "success",
    "started_at": "2022-08-18T12:26:43Z",
    "completed_at": "2022-08-18T12:26:45Z",
    "name": "measure",
    "steps": [
      {
        "name": "Set up job",
        "status": "completed",
        "conclusion": "success",
        "number": 1,
        "started_at": "2022-08-18T12:26:43.000Z",
        "completed_at": "2022-08-18T12:26:43.000Z"
      },
      {
        "name": "Run echo \"Hello, I am a Test!\"",
        "status": "in_progress",
        "conclusion": null,
        "number": 2,
        "started_at": "2022-08-18T12:26:43.000Z",
        "completed_at": null
      }
    ],
    .
    .
    .
 ```
 
(Notice the second step is "in_progress" despite the action being completed)

**Comment by airtower-luna** [comment]
That is odd, you might want to [send a bug report to support](https://support.github.com/contact/bug-report).

**Reply by Ari-Kishon** [reply]
Already Did :) 

Will update once i receive a reply
